File: Code/App/v2/Ward/ViewModels/PatientLogViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Messages;
using Messages.Models;

namespace Ward.ViewModels
{
    public class PatientLogViewModel
    {
        public int PatientDieseaseId { get; set; }
        public int PatientId { get; set; }
        public string Comment { get; set; }
        public DateTime When { get; set; }
        public ExaminationTypeEnum.ExaminationType ExaminationType { get; set; }
        public string ExaminationName { get { return ExaminationTypeEnum.GetName(ExaminationType); } }
        public LogTypeEnum.LogType LogType { get; set; }
    }
}

File: Code/App/separateDB/Ward/Scripts/App/Models/PatientLogViewModel.js
function PatientLogViewModel(data) {
    var self = this;

    this.PatientId = ko.observable(data.PatientId);
    this.PatientDieseaseId = ko.observable(data.PatientDieseaseId);
    this.Comment = ko.observable(data.Comment);
    this.When = ko.observable(data.When);
    this.ExaminationType = ko.observable(data.ExaminationType);
    this.LogType = ko.observable(data.LogType);

    this.ExaminationName = ko.computed(function () {
        return getExaminationTypeName(self.ExaminationType());
    }, this);

    this.ExaminationCssClassName = ko.computed(function () {
        return getExaminationTypeName(self.ExaminationType()) + "-" + getLogTypeName(self.LogType());
    }, this);

    this.LogName = ko.computed(function () {
        return getLogTypeName(self.LogType());
    }, this);

};